qemu/tests/cfg/ksm_base.cfg

```diff
@@ -5,7 +5,7 @@
     requires_root = yes
     setup_ksm = yes
     shared_mem = 1024
-    query_cmd = "cat /sys/kernel/mm/ksm/pages_sharing"
+    xquery_cmd = "cat /proc/QEMU_PID/ksm_merging_pages"
```

Comment:
I would suggest keeping the parameter name, as we are not sure if any other test case calls it as sub-test and resets the cmd. Also, please also check this parameter in KAR.